packages/web-react/src/components/Grid/README.md

```diff
@@ -0,0 +1,28 @@
+# Grid
+
+Use Grid to build multiple column layouts. This Grid works on twelve column system, and it contains variants of 12, 6, 4, 3, 2, and 1 column for each breakpoint. And additionally one centered narrow column.
+
+```jsx
+<Grid cols={6} desktop={4} tablet={3} mobile={2} narrow>
+  <span>col 1</span>
+  <span>col 2</span>
+  <span>col 3</span>
+  <span>col 4</span>
+  <span>col 5</span>
+  <span>col 6</span>
+</Grid>
+```
+
+## Available props
+
+| Name          | Type        | Description                          |
+| ------------- | ----------- | ------------------------------------ |
+| `cols`        | number      | Number of columns to use.            |
```

Comment:
Type `number` is just part of the truth. I believe listing just the available numbers here will lead to fewer developers' confusions 🙂.

Reply:
Added enumeration.